.ruby-version

```diff
@@ -1 +1 @@
-2.2.2
+2.3.6
```

Comment:
🎉

Reply:
Yeah I forgot to ask what version of Ruby you target.